### ISSUE TITLE
No varaibel suggestions with partaials

### ISSUE THREAD

**Issue by packytagliaferro** [mrmlnc/vscode-scss#51]
I have my scss set up like so:

```
Base
   |->_variables.scss
Layouts
   |->_header.scss
App.scss
```

App.scss
```
@import "base/variables";
@import "layouts/header";
```

variables.scss
`$color-primary: black;`

Now in `header.scss` if I start typing `$colo` I get nothing. No hint to use `$color-primary`

Is there something I am missing?

**Comment by TzviCons** [comment]
I believe this is handled in #49 and fixed in pre-release 0.7.0

**Comment by octref** [comment]
Duplicate of #49.
